canvas_sdk/utils/json.py

```diff
@@ -0,0 +1,16 @@
+import json
+from datetime import date, datetime
+from enum import Enum
+
+from pydantic import BaseModel
+
+
+class JSONEncoder(json.JSONEncoder):
```

Comment:
If we put encoders in the model config, we don't need a separate class.

Reply:
This file has been removed.